pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/utils/DriverUtils.java

```diff
@@ -38,9 +41,31 @@ public class DriverUtils {
   public static final String CONTROLLER = "controller";
   private static final String LIMIT_STATEMENT_REGEX = "\\s(limit)\\s";
 
+  // SSL Properties
+  public static final String KEYSTORE_TYPE = "keystore.type";
+  public static final String KEYSTORE_PATH = "keystore.path";
+  public static final String KEYSTORE_PASSWORD = "keystore.password";
+  public static final String TRUSTSTORE_TYPE = "truststore.type";
+  public static final String TRUSTSTORE_PATH = "truststore.path";
+  public static final String TRUSTSTORE_PASSWORD = "truststore.password";
+
   private DriverUtils() {
   }
 
+  public static SSLContext getSSLContextFromJDBCProps(Properties properties) {
```

Comment:
why do we need one additional sslContext parser when TlsUtils already has one? (otherwise we need to make sure the static keyword here matches TlsUtils ones)

Reply:
done. Just wrapping Properties in MapConfiguration worked.